src/data/polynomial/ring_division.lean

```diff
@@ -468,38 +468,50 @@ calc ((roots ((X : R[X]) ^ n - C a)).card : with_bot ℕ)
       ≤ degree ((X : R[X]) ^ n - C a) : card_roots (X_pow_sub_C_ne_zero hn a)
   ... = n : degree_X_pow_sub_C hn a
 
-lemma le_root_multiplicity_map {K L : Type*} [comm_ring K]
-  [comm_ring L] {p : K[X]} {f : K →+* L} (hf : function.injective f) (a : K) :
+section
+
+variables {K L : Type*} [comm_ring K] [comm_ring L]
```

Comment:
If this are not used one million times, can you call them `A` and `B` (or `R` if there is already a ring floating around)? I know it looks silly, but I think it really helps readability.

Reply:
Most of the file uses `R` but it comes with a `[is_domain R]` among the `variables`, while the first two lemmas don't require it, so I can't change `K` to `R`. So I am changing `K` and `L` to `A` and `B` instead, as you suggest. If I am able to remove more `is_domain` hypothesis in the future, then I think we should remove `[is_domain R]` from the `variables`, so that we can also use `R` and `S` here.